apps/marketplace/actions/briefActions.js

```diff
@@ -34,21 +26,16 @@ export const loadBrief = briefId => dispatch => {
 
 export const handleBriefResponseSuccess = () => ({ type: BRIEF_RESPONSE_SUCCESS })
 
-export const handleBriefResponseFailure = response => {
+export const handleBriefResponseFailure = response => dispatch => {
   switch (response.status) {
-    case 409:
-      return {
-        type: BRIEF_RESPONSE_DUPLICATE_FAILURE,
-        // eslint-disable-next-line
-        errorMessage: response.data ? response.data.message : 'a user with this email address already exists'
-      }
+    case 409: {
```

Comment:
I prefer to manage side effects through Higher order components (axios). I think it is neater